deploy/database/views.game.sql

```diff
@@ -22,6 +22,7 @@ AS SELECT
     p.name_ingame AS player_name,
     p.autopass,
     p.fire_overshooting,
+    (length(p.vacation_message) > 0) as is_on_vacation,
```

Comment:
White space gets stripped out, so it won't be a whitespace option. There'a a few subtle places to catch whether a user is on vacation: if they look at their own profile, the Vacation Message will appear only if they are on vacation. Also, a user can see their own vacation status when they're playing a game. I started thinking about putting vacation up on the top ("You are logged in as USER. You are on vacation") but that was starting to become difficult. I'm happy to entertain suggestions!